src/scala/com/github/johnynek/bazel_deps/Resolver.scala

```diff
@@ -57,9 +58,15 @@ class Resolver(servers: List[MavenServer], resolverCachePath: Path) {
   /**
    * Here is where the IO happens
    */
-  private def request(m: MavenCoordinate): CollectResult = {
+  private def request(m: MavenCoordinate, ml: Model): CollectResult = {
     val collectRequest = new CollectRequest()
-    collectRequest.setRoot(new Dependency(new DefaultArtifact(m.asString), ""))
+    val ex = ml.dependencies.excludes(m.unversioned)
+    val exclusions = new util.ArrayList[Exclusion]()
+    for (elem <- ex){
+      val exclusion = new Exclusion(elem.group.asString, elem.artifact.asString, "", "jar")
+      exclusions.add(exclusion)
+    }
```

Comment:
You can replace L64-L68 with a `map` over `ex`. E.g.:
`val exclusions = ex.map(elem => new Exclusion(elem.group.asString, elem.artifact.asString, "", "jar"))`